### DISCUSSION TITLE
Start program does not appear in the dock

### DISCUSSION THREAD

**Discussion by Cardellino** [linuxdeepin/developer-center#3332] [Q&A | 问答板块]
The programs that were started are not visible icons in the bar!
For example Google Earth, Chrome, Libreoffice
See photo....
[Bildschirmaufnahme_Auswahlbereich_20220726173055](https://user-images.githubusercontent.com/73848158/181048452-5daac85b-cb0a-458a-8eda-179ee49ce86b.png)

System Arch Linux 
Kernel 5.18.14
Deepin 20.2.2
Deepin-WM

Thank you in advance for your help

**Comment by BLumia** [comment]
I cannot reproduce this issue on my machine so we need more information to reproduce it. I got Chromium (not Chrome) and Libreoffice installed from the official Arch Linux repo and the icon on taskbar looks fine.

Where are these apps installed from? are these installed via flatpak? Are the icon looks fine in the launcher? If it is fine in launcher, what if use right clicks and select "Send to desktop" and check if the icon is right in desktop?

btw, you can use the [issue board](https://github.com/linuxdeepin/developer-center/issues) for bug report since it has an issue template for you to fill :)

**Reply by Cardellino** [reply]
Good morning Blumia,
All the programs I install are from the arch linux repository mirrorlist.
After the update from 07/25/22, the programs that were started are not displayed in the bar. You're just not there.
For example, if I start Terminal and then send the window (minus) in the background, the program is not displayed in the dock...
Video....
[Aufnehmen_Auswahlbereich_20220727084941](https://user-images.githubusercontent.com/73848158/181181229-aa9db7a0-1872-482a-8b19-52cdbecb0cb2.gif)

**Reply by Cardellino** [reply]
Here is a coredumpctl......

```
coredumpctl gdb

Stack trace of thread 8776:
                #0  0x00007fbad8113814 XDisplayString (libX11.so.6 + 0x2b814)
                #1  0x00007fbad5059a1d __vaDriverInit_1_13 (nvidia_drv_video.so + 0xda1d)
                #2  0x00007fbacafd4c8a n/a (libva.so.2 + 0x3c8a)
                #3  0x00007fbacafdabe1 vaInitialize (libva.so.2 + 0x9be1)
                #4  0x00007fbad39369f7 n/a (libxul.so + 0x85369f7)
                #5  0x00007fbad3937609 n/a (libxul.so + 0x8537609)
                #6  0x00007fbacf8b73de n/a (libxul.so + 0x44b73de)
                #7  0x00007fbacf8b51e5 n/a (libxul.so + 0x44b51e5)
                #8  0x00007fbacf8a8125 n/a (libxul.so + 0x44a8125)
                #9  0x00007fbacf8a73ed n/a (libxul.so + 0x44a73ed)
                #10 0x0000563e19538912 n/a (firefox + 0x49912)
                #11 0x00007fbad9e29290 n/a (libc.so.6 + 0x29290)
                #12 0x00007fbad9e2934a __libc_start_main (libc.so.6 + 0x2934a)
                #13 0x0000563e1954c5b5 _start (firefox + 0x5d5b5)
                ELF object binary architecture: AMD x86-64
Core was generated by `/usr/lib/firefox/firefox'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x00007fbad8113814 in XDisplayString () from /usr/lib/libX11.so.6
(gdb) bt
#0  0x00007fbad8113814 in XDisplayString () at /usr/lib/libX11.so.6
#1  0x00007fbad5059a1d in __vaDriverInit_1_13 () at /usr/lib/dri/nvidia_drv_video.so
#2  0x00007fbacafd4c8a in  () at /usr/lib/libva.so.2
#3  0x00007fbacafdabe1 in vaInitialize () at /usr/lib/libva.so.2
#4  0x00007fbad39369f7 in  () at /usr/lib/firefox/libxul.so
#5  0x00007fbad3937609 in  () at /usr/lib/firefox/libxul.so
#6  0x00007fbacf8b73de in  () at /usr/lib/firefox/libxul.so
#7  0x00007fbacf8b51e5 in  () at /usr/lib/firefox/libxul.so
#8  0x00007fbacf8a8125 in  () at /usr/lib/firefox/libxul.so
#9  0x00007fbacf8a73ed in  () at /usr/lib/firefox/libxul.so
#10 0x0000563e19538912 in  ()
#11 0x00007fbad9e29290 in  () at /usr/lib/libc.so.6
#12 0x00007fbad9e2934a in __libc_start_main () at /usr/lib/libc.so.6
#13 0x0000563e1954c5b5 in _start ()
```

**Reply by BLumia** [reply]
The coredump seems unrelated. It that affacted by the `07/25/22` update? I think the only difference between my installation and the official repo is I am using a `-git` version of `startdde` package...

**Reply by BLumia** [reply]
[startdde-git-0.0.0+r+ga0bce19-1-x86_64.pkg.zip](https://github.com/linuxdeepin/developer-center/files/9196150/startdde-git-0.0.0%2Br%2Bga0bce19-1-x86_64.pkg.zip)

For the record, my `-git` package is uploaded here so you can try if this is really the case. It's only for testing purpose btw, and if you decided to try, a gsettings config value (can be edit via `dconf editor`) at `/com/deepin/dde/startdde/wm-cmd` may need to change to `deepin-wm`.

**Reply by Cardellino** [reply]
Hello, I installed package startdde-git-0.0.0+r+ga0bce19-1-x86_64.pkg. After a reboot I started the program sudo dconf-editor via terminal.
When changing to deepin-wm, the error comes dconf-editor:3427): dconf-WARNING **: 10:48:04.266: failed to commit changes to dconf.

I can't change the value...it won't accept that...
I will reinstall the system.
Many thanks for the support Blumia..:):)+

**Reply by BLumia** [reply]
> I can't change the value...it won't accept that...

You need to turn off the "use default value" switch, and you don't need to use `sudo` to run dconfig-editor. If you want to rollback, you can simply install `startdde` from the official repo via `pacman` and the `-git` version will be replaced so you don't need to reinstall the whole system :)

**Reply by Cardellino** [reply]
Good morning Blumia,
I reset everything and swapped the startdde-git for me startdde...
Now it works again..
Resetting to "default value" helped...
Thank you again for your support.
Have a nice day.

**Reply by BLumia** [reply]
Oh, glad it works, so does it means using the `startdde` from Arch repo will cause the problem you've mentioned?

**Reply by Cardellino** [reply]
many updates have been made in the last few days...I can only say that it works again after a default value....
In the future update only after a few days.. :) :)
Thanks again...

**Reply by BLumia** [reply]
Oh okay. btw still need to say the `-git` package is only for testing, it's recommended to switch back to regular `startdde` package from the Arch repo so it will get regular updates when it has one :)

**Reply by Cardellino** [reply]
already done...
